lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -28,6 +28,8 @@ Gem::Specification.new do |spec|
       "public gem pushes."
   end
 
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` default will base this on all the files checked in to git.
```

Comment:
> The `git ls-files -z` default will base this on all the files checked in to git.

This doesn't read that well when i say it out loud. How about we change this to something like:

```
The `git ls-files -z` loads the files in the RubyGem that have been added into git
```

?

Reply:
sure